src/app/location/location.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from '@angular/core';
+import { OpenWeatherService } from 'src/app/services/open-weather.service';
+
+@Component({
+  selector: 'app-location',
+  templateUrl: './location.component.html',
+  styleUrls: ['./location.component.scss'],
+})
+export class LocationComponent {
+  lat;
+  lon;
+  city = 's';
+
+  constructor(protected owService: OpenWeatherService) {}
+
+  getUserLocation(): void {
+    // get Users current position
+
+    if (navigator.geolocation) {
+      navigator.geolocation.getCurrentPosition(
+        (position) => {
+          this.lat = position.coords.latitude;
+          this.lon = position.coords.longitude;
+          console.log(`position: ${this.lat} , ${this.lon}`);
+        },
+        (decline) => {
+          console.log('Permission declined');
```

Comment:
As well as no geolocation supported we probably would need to show this to a user. Please add TODO comment